torch/csrc/jit/tensorexpr/bounds_overlap.cpp

```diff
@@ -114,6 +114,13 @@ OverlapKind overlaps(const IndexBounds& a, const IndexBounds& b) {
 }
 
 std::vector<Bound> subtractBound(Bound a, Bound b, OverlapKind overlap) {
+  if (overlap == NoOverlap) {
```

Comment:
Do we even need to have these two versions of the function? Why don't we inline this one into the second one? Do we expect that the version with the "overlap" argument is independently useful too? In other words, isn't overlap handling an internal detail that should not be exposed in the API?

Reply:
Sure, good point. Removed the other overload.